src/index.html

```diff
@@ -1,13 +1,343 @@
+    >
+    <link href="https://fonts.googleapis.com/css2?family=Space+Grotesk&family=Space+Mono&display=swap" rel="stylesheet">
   </head>
-  <body>
-    <h1>Landing Page</h1>
+    
+  <body class="page__body">
+    <header class="header">
+      <div class="container">
+        <div class="header__content">
+          <div class="top-bar">
+            <a href="#" class="top-bar__logo-link">
+              <img
+                class="top-bar__logo"
+                src="images/logo.svg"
+                alt="Nothing logo"
+              >
+            </a>
+            <div class="top-bar__icons">
+              <a href="tel:+1 234 5555-55-55" class="icon icon--phone"></a>
+              <a href="#menu" class="icon icon--menu"></a>
+            </div>
+          </div>
+        
+          <div class="header__bottom--title">
+            Bring joy back<br> to the everyday
+          </div>
+        </div>
+      </div>
+    </header>
+
+    <aside class="page__menu menu" id="menu">
+      <div class="container">
+        <div class="top-bar menu__top">
+          <a href="#" class="top-bar__logo-link">
+            <img
+              class="top-bar__logo"
+             src="images/logo.svg"
+              alt="Nothing logo"
+            >
+          </a>
+          <div class="header__icons">
+            <a href="#" class="icon icon--close"></a>
+          </div>
+        </div>
+        
+        <div class="menu_bottom">
+          <nav class="nav menu__nav">
+            <ul class="nav__list">
+              <li class="nav__item">
+                <a href="#recommended" class="nav__link">Products</a>
+              </li>
+              <li class="nav__item">
+                <a href="#categories" class="nav__link">Store</a>
+              </li>
+              <li class="nav__item">
+                <a href="#footer" class="nav__link">Login</a>
+              </li>
+              <li class="nav__item">
+                <a href="#" class="nav__link">Cart(0)</a>
+              </li>
+            </ul>
+          </nav>
+        
+          <a
+            href="tel:+1 234 5555-55-55"
+            class="menu__phone-number menu__bottom"
+          >
+            +1 234 5555-55-55
+          </a>
+        
+          <a
+            href="tel:+1 234 5555-55-55"
+            class="menu__call-to-order menu__bottom"
+          >
+            Call to order
+          </a>
+        </div>
+      </div>
+    </aside>
+
+    <main class="main">
+      <div class="container">
+        <section class="recommended" id="recommended">
+          <h2 class="section-title">Recommended</h2>
+          <div class="recommended__products">
+            <article class="product">
+              <a href="#">
+                <img
+                  src="images/Phone(1).png"
+                  alt="Phone(1)"
+                  class="product__photo"
+                >
+              </a>
+              <h3 class="product__title">Phone (1)</h3>
```

Comment:
It is better to give a more meaningful description for `alt`, for example `Recommened product Phone 1`.
Fix this in all code.